app/pathogen/arbovirus/dashboard/(visualizations)/chart-arbovirus-dropdown.tsx

```diff
@@ -0,0 +1,95 @@
+import { useContext, useMemo, useState, useCallback } from "react";
```

Comment:
This is mostly copy-paste from the WHO region and Age group table. I wanted to make that code reusable across both components and quarantine the gross things I had to do to make this work with all of the edge cases.